quic/s2n-quic-platform/src/message/mmsg.rs

```diff
@@ -57,6 +67,11 @@ impl MessageTrait for mmsghdr {
         self.msg_hdr.replicate_fields_from(&other.msg_hdr)
     }
 
+    #[inline]
+    fn validate_replication(source: &Self, dest: &Self) {
+        libc::msghdr::validate_replication(&source.msg_hdr, &dest.msg_hdr)
```

Comment:
I dont see validate_replication in the docs. Am I missing something? https://docs.rs/libc/latest/libc/struct.msghdr.html

Reply:
this is impled by the MessageTrait